ClusterTest/src/main/scala/net/snowflake/spark/snowflake/ClusterTest.scala

```diff
@@ -30,6 +37,10 @@ object ClusterTest {
 
   // Driver function to run the test.
   def main(args: Array[String]): Unit = {
+    // If there are more than 2 arguments, we're running in the stress test environment
+    val envType: ClusterTest =
```

Comment:
It should be better to add a new argument to distinction the 2 kinds of run, instead of argument count.
lets say, the 3rd argument can be "githubtest" or "stresstest".

Reply:
If we do this, please update the script in ClusterTest.

Reply:
Done.